app/External/Slack/ValidatesSlack.php

```diff
@@ -31,6 +32,11 @@ public function isSignatureValid(Request $request, string $signingSecret): bool
 
     public function isCertificateValid(Request $request): bool
     {
+        if (config('app.env') == 'local') {
+            Log::warning('APP_ENV is local, skipping mTLS check for Slack');
+            return true;
+        }
```

Comment:
Is there a better way to do this? ngrok has an mTLS module but it's kinda a pain.

Reply:
`App::environment('local')` for the command to check if you're running locally. Personally, if you're testing locally there's not too much reason to verify the Slack cert unless you leave that tunnel open.